src/SFA.DAS.EAS.DbMaintenance.UnitTests/app.config

```diff
@@ -28,7 +28,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Microsoft.IdentityModel.Clients.ActiveDirectory" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.16.0.14" newVersion="3.16.0.14" />
+        <bindingRedirect oldVersion="0.0.0.0-3.13.8.999" newVersion="3.13.8.999" />
```

Comment:
Again why are we downgrading the version?

Reply:
Fixed